include/zephyr/bluetooth/conn.h

```diff
@@ -2064,6 +2285,31 @@ struct bt_conn_cb {
 				const struct bt_conn_le_subrate_changed *params);
 #endif /* CONFIG_BT_SUBRATING */
 
+#if defined(CONFIG_BT_SHORTER_CONNECTION_INTERVALS)
```

Comment:
```suggestion
#if defined(CONFIG_BT_SHORTER_CONNECTION_INTERVALS) || defined(__DOXYGEN__)
```

Need this part to enable Doxygen for conditionally compiled callbacks and fields